dotnet/src/Connectors/Connectors.Memory.AzureCosmosDBNoSQL/AzureCosmosDBNoSQLMemoryStore.cs

```diff
@@ -332,7 +339,7 @@ FROM x
             {
                 if (memoryRecord.SimilarityScore >= minRelevanceScore)
                 {
-                    yield return (memoryRecord, memoryRecord.SimilarityScore);
+                    yield return (memoryRecord, 1 - memoryRecord.SimilarityScore);
```

Comment:
I'm following up on whether we should always do this, or if it depends on the configured distance function in Cosmos DB.

Reply:
@jcodella - any thoughts about this? Do we need to remember the distance function passed in and have a different operation here?

Reply:
Restricted to just `Cosine` and corrected this calculation.  @jcodella does that seem reasonable.